stylesheets/components/alert.css

```diff
@@ -0,0 +1,75 @@
+.alert {
+  width: 100%;
+  font-size: 1.6rem;
+  font-weight: 600;
+  display: flex;
+  align-items: center;
+  gap: 16px;

```

Comment:
I think you should use 1rem here instead of 16px as px don't scale

Reply:
Here I used pixel intentionally bcoz don't want to change the gap between icon and text.